app/assets/javascripts/app/views/gallery_view.js

```diff
@@ -0,0 +1,44 @@
+app.views.Gallery = app.views.Base.extend({
+  pictures: function(){
+    return this.$el.find("a.gallery-picture");
+  },
+
+  initialize: function(){
+    var self = this;
+    console.log(this.$el.find("a.gallery-picture").length + " pictures found");
+
+    this.pictures().click(function(event){
```

Comment:
I think you can move this instead to a events hash. http://backbonejs.org/#View-events